build_local.sh

```diff
@@ -11,7 +11,8 @@ docker ps
 # Cleanup from previous build
 rm -rf /tmp/dcos_build_venv
 
-# Force Python stdout/err to be unbuffered.
+# Force Python stdout/err to be unbuffered to have immediate
+# feedback in e.g. a TeamCity build environment.
 export PYTHONUNBUFFERED="notemtpy"
```

Comment:
:question:  `PYTHONUNBUFFERED` just needs to be a non-empty string. If you are in a house-cleaning mood, consider fixing `s/notemtpy/notempty/`

Reply:
Let's take care of this in a follow-up :)